### PR TITLE
Refactor/harmonize chevrons

### PR DESCRIPTION
**Pull request by 0xLodz**
Since we are using two kinds of chevrons in multiple part of the application, we need to keep them in their simple form. This PR prepares the creation of a collapsible component that will use them. The collapsible component is a reusable component needed for filters, articles, menus, and several other elements of the design system.

### CODE REVIEW

File: src/ui/component/toolbar/toolbar.tsx [hunk 1]
```diff
@@ -133,11 +131,11 @@ const WalletButton = ({
             `okp4-dataverse-portal-wallet-expand ${isWalletMenuOpen ? 'rotate-down' : 'rotate-up'}`
           )}
         >
-          <Icon name={`expand-down-${theme}`} />
+          <Icon name={`chevron`} />
```
Comment (lolottetheclash) [inline src/ui/component/toolbar/toolbar.tsx:134]: You forgot the `{}` !

**Review by lolottetheclash — CHANGES_REQUESTED** [review]
Nice refactor, thanks !
A tiny oversight to fix and I'm good to go!

File: src/ui/component/toolbar/toolbar.tsx (hunk 1 above)
Comment (0xLodz) [reply src/ui/component/toolbar/toolbar.tsx]: Fixed! Thanks

**Review by lolottetheclash — APPROVED** [review]
👍

**Review by fredericvilcot — APPROVED** [review]

**Pull request merged**